test/expected/telemetry.out

```diff
@@ -332,33 +326,15 @@ SELECT json_object_field(get_telemetry_report()::json,'instance_metadata');
  {"cloud": "ci"}
 (1 row)
 
---create a continuous agg
-CREATE TABLE device_readings (
-      observation_time  TIMESTAMPTZ       NOT NULL
-);
-SELECT table_name FROM create_hypertable('device_readings', 'observation_time');
-   table_name    
------------------
- device_readings
-(1 row)
-
-CREATE VIEW device_summary
-WITH (timescaledb.continuous) AS
-SELECT
-  time_bucket('1 hour', observation_time) as bucket,
-  min(observation_time)
-FROM
-  device_readings
-GROUP BY bucket;
-SELECT json_object_field(get_telemetry_report()::json,'num_continuous_aggs');
+SELECT json_object_field(get_telemetry_report()::json,'num_hypertables');
  json_object_field 
 -------------------
- "1"
+ "0"
```

Comment:
should at least have 1 hypertable here, or print num_hypertables, create hypertable and then print it again